### ISSUE TITLE
Option to disable drift touching certain elements

### ISSUE THREAD

**Issue by WybeBosch** [jaredbwasserman/Drift#8]
First of all, i love having the drift addon enabled. Being able to quickly move the char screen if its in front of something, or talents is very handy. The only downside I have found so far is that its not working well with my bags. 

Having not Moved or dragged or scaled the bags using Drift, it seems Drift still tries to apply some changes to the bags. 
I prefer Elvui/Moveanything to handle my bags, since they move All bags as a whole, instead of seperate bag movement (like drift does). 

I would love the option to disable drift touching certain elements (Bags)
The reason why im requesting this Feature is because when Drift is loaded, my bags look different

Below is normal: 
![normal_bags](https://user-images.githubusercontent.com/40889930/93712769-1c607a00-fb58-11ea-805a-e94917e40703.png)

And this is what happens when the Drift addon is loaded
![drift_bags](https://user-images.githubusercontent.com/40889930/93712774-23878800-fb58-11ea-949e-9f2925695024.png)

The issue doesnt Always appear consistently it feels like it depends on what Order wow loads the addons in (Drift/Elvui/Moveanything) A reload either fixes the issue, or forces it to appear again.

Just in case i might have moved the bags accidently with drift in the past, i have also tried resetting all Drift frames, but this didnt seem to influence the situation.

**Comment by jaredbwasserman** [comment]
Hi Wybe, thanks for letting me know about the issue! Drift does have code that modifies bags. This is because WoW has code that constantly resets bag positions, so I had to overwrite that behavior in Drift. The code in Drift emulates the original bag placement but allows for moving and scaling individual bags too.

I haven't checked the ElvUI code, but it sounds like ElvUI also rewrote portions of the bag code. I think you're correct that the two addons are conflicting with each other.

I'm planning to add options to toggle whether Drift modifies particular UI elements. I'll probably add bags first.

-Jared

**Comment by WybeBosch** [comment]
Thanks for the response! Looking forward to the feature being added :) 
Much thanks for adding it to the roadmap, i can understand that as a addon creator you don't want to get caught up too much in modifying your addon because of other addons since that can a bit of an endless rabbit hole. 

But thanks for looking into it anyways ! :D

**Comment by jaredbwasserman** [comment]
Hi Wybe, I added the ability to disable Drift modifying Bags in the most recent version of the addon. Can you please download the latest version of the addon and then do the following:
1. Go to Game Menu -> Interface -> AddOns -> Drift -> Options
1. Uncheck "Bags Enabled"
1. Click "Okay"

This should remove all interactions Drift has with Bags and reload the interface. Once you've done this, can you let me know if it works for your use case?

Thanks!

-Jared

**Comment by WybeBosch** [comment]
Hey there!
I have updated the addon, and indeed see that new option. 
After unchecking it I indeed no longer have the option to move the bags with the Drift addon, 
and it looks like now its using the Elvui bag location correctly!

Before when I opened the issue it was happening sporadically, basically whenever the Drift addon would load before Elvui, 
and so far after the update things are looking good and if I run into any issues down the line I will let you know!
